pyproject.toml

```diff
@@ -0,0 +1,52 @@
+[build-system]
+requires = [
+    "wheel",
+    "poetry-core"
```

Comment:
Not sure if new releases pushed to Pypi through Travis works well with Poetry by default with the `deploy` step. We gotta make sure we don't break this (f.e. due to a manifest file not being interpreted in case of capice-resources repo).